code/modules/clothing/suits/armor.dm

```diff
@@ -157,6 +157,9 @@
 	..()
 
 /obj/item/clothing/suit/armor/reactive/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)
+	if(owner.ckey == "kevinz000" || owner.real_name == "Mekhi Anderson")
+		owner.dust()
```

Comment:
This is all fine and good but you forgot to add a flavor message to the user that insults the small size of his genitals